ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -350,6 +350,16 @@ GeneDetails.propTypes = {
   rnaSeqData: PropTypes.object,
 }
 
+const GeneSearchLinkWithPopup = props => (
+  <Popup
+    trigger={
+      <GeneSearchLink {...props} />
+    }
+    content="Search for all variants with AF < 10% in this gene present in any affected individual"
```

Comment:
`AF < 10%` is just the default value. It is not true after user changes it. Add `by default` to the sentence?

Reply:
it is going to open up a search page with search results with those exact filters applied, its not pre-populating a form you can adjust. Also this was explicitly asked for in the ticket

Reply:
I see.